src/main/java/com/github/splendor_mobile_game/websocket/handlers/reactions/CreateRoom.java

```diff
@@ -29,10 +29,12 @@ public CreateRoom(int connectionHashCode, UserMessage userMessage, Messenger mes
     public static class RoomDTO {
         public String name;
         public String password;
+        public UUID uuid;
```

Comment:
That's not how it should work. What you are now saying is that you require the user to provide the UUID of the room when they are creating it, while at the same time, you are generating a random UUID on line 139 when constructing the room.

I propose the following:
- The user won't provide the UUID when creating the room.
- The server will generate the UUID of the room.
- The server will send back to the client the room ID it has generated.